lib/datadome/configuration.rb

```diff
@@ -11,9 +11,10 @@ def initialize
       @include_matchers = []
       @monitor_mode = false
       @expose_headers = false
+      @faraday_open_timeout = 1
```

Comment:
:thinking: WDYT of removing the `faraday_` prefixes here? We shouldn't care which http client the gem is using.

Reply:
it will be a bit lighter, go for it. I thought of replacing it with `http_client_` but it's too long.